kolibri/core/content/serializers.py

```diff
@@ -437,7 +438,16 @@ class Meta:
 
     @property
     def channel_stats(self):
-        return self.context["channel_stats"]
+        channel_stats = self.context["channel_stats"]
+        if connection.vendor == "postgresql":
```

Comment:
maybe for a later date, but I've grepped this line `connection.vendor == "postgresql"` and it appears in 10 different places. maybe we can extract it to a utility later.